website/content/docs/sync/vercelproject.mdx

```diff
@@ -4,4 +4,81 @@ page_title: Vercel Project - Secrets Sync Destination
 description: The Vercel Project destination syncs secrets from Vault to Vercel.
 ---
 
-# Vercel Project
+# Vercel Project environment variables
+
+The Vercel Project sync destination allows Vault to safely synchronize secrets as Vercel environment variables.
+This is a low footprint option that enables your applications to benefit from Vault-managed secrets without requiring them
+to connect directly with Vault. This guide walks you through the configuration process.
+
+Prerequisites:
+* Ability to read or create KVv2 secrets
+* Ability to create Vercel tokens with access to modify project environment variables
+* Ability to create sync destinations and associations on your Vault server
+
+## Setup
+
+1. If you do not already have a Vercel token, navigate [your account settings](https://vercel.com/account/tokens) to
+   generate credentials with the necessary permissions to manage your project's environment variables.
+
+1. Next you need to locate your project ID. It can be found under the `Settings` tab in your project's overview page.
```

Comment:
Is it possible to pass the project name and have the lib take care of the translation into an ID so it is easier to use? The project ID is not super discoverable.